src/handlers/sqs-indexer/handle-bridge-out-event.ts

```diff
@@ -0,0 +1,223 @@
+// import {TransactionReceipt, TransactionResponse} from '@ethersproject/abstract-provider'
```

Comment:
comment this file back in. I get that its the older stuff, but at least it can still be here until we delete it.